packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-disabled.tsx

```diff
@@ -0,0 +1,100 @@
+                                        key={0}
+                                    />,
+                                    <a
+                                        className='link two-factor__link'
+                                        href='https://github.com/google/google-authenticator/wiki#implementations'
+                                        target='_blank'
+                                        rel='noopener noreferrer'
+                                        key={1}
+                                    />,
+                                ]}
+                            />
+                        }
+                    >
+                        <div className='two-factor__qr'>
+                            {is_qr_loading ? (
+                                <Loading is_fullscreen={false} />
+                            ) : (
+                                <React.Fragment>
+                                    {qr_secret_key && (
+                                        <div className='two-factor__qr--wrapper'>
+                                            <QRCodeSVG value={qr_secret_key} />
+                                        </div>
+                                    )}
+
+                                    {secret_key && (
+                                        <React.Fragment>
+                                            <Text as='h4' size='xs' align='center' className='two-factor__qr--message'>
+                                                <Localize i18n_default_text='If you are unable to scan the QR code, you can manually enter this code instead:' />
+                                            </Text>
+                                            <div className='two-factor__qr--code' data-testid='dt_2fa_clipboard'>
+                                                <Text size='xs'>{secret_key}</Text>
+                                                <Clipboard
+                                                    text_copy={secret_key}
+                                                    info_message={localize('Click here to copy key')}
+                                                    success_message={localize('Key copied!')}
+                                                    className='two-factor__qr--clipboard'
+                                                />
+                                            </div>
+                                        </React.Fragment>
+                                    )}
+                                </React.Fragment>
+                            )}
+                        </div>
+                    </Timeline.Item>
+                    <Timeline.Item
+                        item_title={
+                            <Localize i18n_default_text='Enter the authentication code generated by your 2FA app:' />
+                        }
+                    >
+                        <div data-testid='dt_digitform_2fa_disabled'>
```

Comment:
do we need here additional <div>?
Timeline.Item is a <div>

Reply:
Yes. We need it according to the design :)

Reply:
but TimeLine.Item is already div. sorry, the elements I mentioned was hidden in previous message :)
so here we have div inside div

Reply:
`<div data-testid='dt_digitform_2fa_disabled'>` is introduced to test digitform specifically